src/star-rating/star-rating.svelte

```diff
@@ -0,0 +1,63 @@
+<script>
+  import { createEventDispatcher } from 'svelte';
+  import Star from './star.svelte';
+  import range from '../_utils/range.js';
+  import classes from '../_utils/classes.js';
+  import ripple from '../_utils/ripple.js';
+  import eventsAction from '../_utils/events.js';
+
+  let _class = null;
+  export { _class as class };
+  export let starClass = null;
+
+  export let max = 5;
+  export let value = null;
+  export let name;
+  export let events = null;
+
+  const ARROW_LEFT_CODE = 37;
+  const ARROW_UP_CODE = 38;
+  const ARROW_RIGHT_CODE = 39;
+  const ARROW_DOWN_CODE = 40;
+
+  function reverseArrowKeys(event) {
+    switch (event.keyCode) {
```

Comment:
`event.keyCode` has been deprecated. Better use [`KeyboardEvent.code`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/code) or [`KeyboardEvent.key`](https://developer.mozilla.org/en-US/docs/Web/API/KeyboardEvent/key). That would have the added benefit of using the string representation right away instead of the ugly `const`s